waspc/cli/Main.hs

```diff
@@ -89,6 +96,28 @@ printUsage =
 printVersion :: IO ()
 printVersion = putStrLn $ showVersion version
 
+-- generat the bash completion script
+generateBashCompletionScript :: IO ()
+generateBashCompletionScript =
+  getDataFileName "Cli/bash-completion" >>= readFile >>= putStr
+
+-- return the bash completion instruction
+printBashCompletionInstruction :: IO ()
+printBashCompletionInstruction =
+  putStrLn $
+    unlines
+      [ "Run the following command to generate bash completion script for wasp on your machine:",
+        "",
+        "wasp completion:generate > <your-chosen-directory>/wasp-completion",
+        "",
+        "After that, depending on your system, you will need to edit your bash profile:",
+        "- on MacOS (OSX): you will normally want to edit ~/.bashrc",
+        "- on Linux: you will normally want to edit ~/.bash_profile",
+        "and add this line:",
+        "  source <your-chosen-directory>/wasp-completion",
+        "then reset your terminal session."
+      ]
+
```

Comment:
I would move this logic to Command/BashCompletion.hs, so it sticks with other bash completion logic.